plugins/module_utils/mh/base.py

```diff
@@ -31,6 +31,11 @@ def __init__(self, module=None):
     def diff_mode(self):
         return self.module._diff
 
+    def do_raise(self, *args, **kwargs):
+        exc_class = kwargs.get("exception_class", _MHE)
```

Comment:
Such constructs are a cause for quite a few security vulnerabilities, where user-provided data suddenly controls execution due to some obscure feature of a library. I would stick to the default class here.
```suggestion
        exc_class = _MHE
```

Reply:
okie